modules/dreamview/frontend/src/renderer/routing_editor.js

```diff
@@ -92,16 +92,20 @@ export default class RoutingEditor {
         }
     }
 
-    sendRoutingRequest(carOffsetPosition, carHeading, coordinates) {
-        if (this.routePoints.length === 0) {
+    sendRoutingRequest(carOffsetPosition, carHeading, coordinates, routingPoints) {
+        if (this.routePoints.length === 0 && routingPoints.length === 0) {
```

Comment:
There are two indent styles in this file: 2 spaces and 4 spaces. Please unify into the former style.

Reply:
thx